webpack.config.js

```diff
@@ -194,7 +204,7 @@ module.exports = function(env = {}, argv = {}) {
       filename: "[name].[chunkhash:6].js",
       path: path.resolve(__dirname, "dist")
     },
-    devtool: mode === "development" ? "cheap-eval-source-map" : "source-map",
+    devtool: false,
```

Comment:
Looks like this is the default. Do we need this?

https://webpack.js.org/configuration/devtool/#devtool

Reply:
Nope! Removed